systemd/ignition-firstboot-complete.service

```diff
@@ -2,18 +2,18 @@
 Description=Mark boot complete
 Documentation=https://github.com/coreos/ignition
 # Note that deriving from Container Linux, there is
-# a tradition that coreos.first_boot=1 is used for PXE/diskless
-# scenarios where it's *always* first boot, so we don't want
-# to try to change `/boot` there.
-ConditionKernelCommandLine=coreos.firstboot=detected
+# a tradition that coreos.first_boot=1 (now ignition.firstboot) 
```

Comment:
Maybe rephrase this to clearify?

Reply:
I think we can possibly just remove this entire comment for now until we "add back" diskless PXE support